src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,31 @@
+package duke.commands;
+
+import duke.exceptions.InvalidInputException;
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents an abstract command in the DailyTasks application.
+ * <p>
+ * Commands are responsible for performing specific actions that manipulate
+ * the task list, interact with the user interface, or modify the storage.
+ * Subclasses must implement the {@link #execute(TaskList, Ui, Storage)} method
+ * to define specific behavior.
+ * </p>
+ */
+public abstract class Command {
```

Comment:
great work creating different classes to handle different tasks!!
Will make your life so easy while testing.
Even I will consider doing so 🙇🏼‍♂️